extensions/amp-ad/0.1/test/test-amp-ad-api-handler.js

```diff
@@ -109,7 +112,32 @@ describe('amp-ad-api-handler', () => {
         requestedHeight: 217,
         type: 'embed-size-changed',
         sentinel: 'amp3ptest' + testIndex,
-      }));
+      })).then(() => {
+        expect(iframe.height).to.equal('217');
+        expect(iframe.width).to.equal('114');
+      });
+    });
+
+    it('should be able to use embed-size API to resize height only', () => {
+      iframe.height = 11;
+      iframe.width = 22;
+      sandbox.stub(adImpl, 'attemptChangeSize', () => {
+        return Promise.resolve();
```

Comment:
https://github.com/ampproject/amphtml/blob/master/extensions/amp-ad/0.1/test/test-amp-ad-api-handler.js#L81

I thought this would be enough. You mean add a rejection test for resizing height only?